Cargo.toml

```diff
@@ -39,7 +39,7 @@ rand_core = { version = "0.6", features = ["std"] }
 rand_chacha = "0.3"
 
 [features]
-default = ["rand"]
+default = ["rand", "alloc"]
```

Comment:
Most of the downstream crates which use this crate don't need the `alloc` feature. It's really just for `rsa`/`dsa`.